keps/sig-apps/1847-autoremove-statefulset-pvcs/README.md

```diff
@@ -305,38 +310,57 @@ In order to update the PVC ownerReference, the `buildControllerRoles` will be up
-    - With `Delete` policies for `whenScaled` and `whenDeleted`
-      1. Create 2 pod statefulset, scale to 1 pod, confirm PVC deleted
-      1. Create 2 pod statefulset, add data to PVs, scale to 1 pod, scale back to 2, confirm PV empty.
-      1. Create 2 pod statefulset, delete stateful set, confirm PVCs deleted.
-      1. Create 2 pod statefulset, add data to PVs, manually delete one pod, confirm pod comes back and PV still has data (PVC not deleted).
-      1. As above, but manually delete all pods in stateful set.
-      1. Create 2 pod statefulset, add data to PVs, manually delete one pod, immediately scale down to one pod, confirm PVC is deleted.
-      1. Create 2 pod statefulset, add data to PVs, manually delete one pod, immediately scale down to one pod, scale back to two pods, confirm PV is empty.
-      1. Create 2 pod statefulset, add data to PVs, perform rolling confirm PVC don't get deleted and PV contents remain intact and useful in the updated pods.
-    - With `Delete` policy for `whenDeleted` only
-      1. Create 2 pod statefulset, scale to 1 pod, confirm PVC still exists,
-      1. Create 2 pod statefulset, add data to PVs, scale to 1 pod, scale back to 2, confirm PV has data (PVC not deleted).
-      1. Create 2 pod statefulset, delete stateful set, confirm PVCs deleted
-      1. Create 2 pod statefulset, add data to PVs, manually delete one pod, confirm pod comes back and PV has data (PVC not deleted).
-      1. As above, but manually delete all pods in stateful set.
-      1. Create 2 pod statefulset, add data to PVs, manually delete one pod, immediately scale down to one pod, confirm PVC exists.
-      1. Create 2 pod statefulset, add data to PVs, manually delete one pod, immediately scale down to one pod, scale back to two pods, confirm PV has data.
-    - Retain: 
-      1. same tests as above, but PVCs not deleted in any case and confirm data intact on the PV.
-    - Pod restart tests:
-      1. Create statefulset, perform rolling update, confirm PVC data still exists.
-    - `--casecade=false` tests.
-1. Upgrade/Downgrade tests
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+#### Unit tests
+
+- `pkg/controller/statefulset`: [test coverage](https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit&include-filter-by-regex=statefulset)
+  - `TestStatefulPodControlUpdatePodClaimForRetentionPolicy`
+  - `TestStatefulPodControlRetainDeletionPolicyUpdate`
+  - `TestStatefulPodControlRetentionPolicyUpdate`
+  - `TestStatefulPodControlRetentionPolicyUpdateMissingClaims`
+  - `TestOrphanedPodsWithPVCDeletePolicy`
+  - `TestStaleOwnerRefOnScaleup`
+  - `stateful_set_control_test`
+     - Added `runTestOverPVCRetentionPolicies` to augment existing tests with
+    coverage for retention policies.
+     - `checkClaimInvarients` extended to test new feature.
```

Comment:
You're still not following the template: https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md?plain=1#L257-L328, ie. here you should have:
```
<package>: <date> - <current test coverage>
```

Reply:
Sorry, I guess I don't understand what the template is trying to do. What is the date referring to?

Reply:
These are updated now as best as I could figure.